cmd/swarmctl/main.go

```diff
@@ -41,7 +41,7 @@ func defaultSocket() string {
 	if swarmSocket != "" {
 		return swarmSocket
 	}
-	return "./swarmkitstate/swarmd.sock"
+	return "/run/swarmd/swarmd.sock"
```

Comment:
I was following `/run/containerd/containerd.sock`. Actually `/var/run` is a symlink to `/run` on my system so the end result is the same.